### PR TITLE
[1.x] Laravel way polymorphic relationship

### PR DESCRIPTION
**Pull request by JohnnyMaynne**

### CODE REVIEW

**Comment by milwad-dev** [comment]
Did breaking changes?

**Comment by JohnnyMaynne** [comment]
> Did breaking changes?

Maybe 😊. I accidentally came across the package and did a PR in 2 minutes and didn't test it. I will find time and try to fix it.

I found that you don't use indexes in the database, which can slow down working with large amounts of data, and you also don't use migration helpers for polymorphic relationships, which slightly breaks the "correct way" that the documentation advises.

It also seems to me that the uuid setting is not particularly necessary since you give the opportunity to publish the migration.

**Comment by milwad-dev** [comment]
Good, so please fix the styleci

**Comment by JohnnyMaynne** [comment]
> Good, so please fix the styleci

I corrected the style

**Comment by JohnnyMaynne** [comment]
and I just realized that I didn’t create a migration, but corrected an existing one, it turns out that it will be incompatible with previous versions

**Comment by JohnnyMaynne** [comment]
so I'll post this soon too, so don't combine it yet

**Comment by JohnnyMaynne** [comment]
seems like everything, but I still haven't tested in the application - I created a new migration in which the columns will be renamed and I also added indexes

File: migrations/2023_12_07_154238_rename_column_and_add_index_attributes_table.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+return new class() extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table(config('laravel-attributes.tables.name', 'attributes'), function (Blueprint $table) {
+            $table->renameColumn('attributable', 'attributable_type');
+            $this->index(['attributable_type', 'attributable_id']);
```
Comment (milwad-dev) [inline migrations/2023_12_07_154238_rename_column_and_add_index_attributes_table.php]: $table is correct

File: migrations/2023_12_07_154238_rename_column_and_add_index_attributes_table.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::table(config('laravel-attributes.tables.name', 'attributes'), function (Blueprint $table) {
+            $table->renameColumn('attributable_type', 'attributable');
+            $this->index(['attributable_type', 'attributable_id']);
```
Comment (milwad-dev) [inline migrations/2023_12_07_154238_rename_column_and_add_index_attributes_table.php]: Also here

**Comment by JohnnyMaynne** [comment]
I apologize for the stupid typo, not a very good decision to do PR on the road

**Pull request closed without merging**